lib/forspell/loaders/c.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+require_relative 'ruby'
+
+module Forspell::Loaders
+  class C < Ruby
```

Comment:
C loader is inherite from Ruby loader?.. Hm.. Maybe it is better to have common functionality as abstract `SourceLoader`, from which C and Ruby are inherited?

Reply:
The necessity is questionable due to `SourceLoader` would be... empty?